src/scene_graph/camera_node.hpp

```diff
@@ -89,6 +89,7 @@ namespace wr
 		float m_shape_amt;
 		int m_aperture_blades;
 		bool m_enable_dof = false;
+		bool m_enable_bloom = false;
```

Comment:
Bloom shouldn't be part of the camera. use the settings for render task system if you want settings. (See hbao+ task)

Reply:
I will resolve this in the bloom pull request as bloom needs a rework anyways.